src/app/auth/settings/settings.constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum SettingsTabs {
+    PROFILE = "profile",
+    ACCOUNT = "admin",
+    APPEARANCE = "appearance",
+    ACCESSIBILITY = "accessibility",
+    NOTIFICATIONS = "notifications",
+    BILLING = "billing",
```

Comment:
I think there was a popup menu item for billing, can we link to here? Please review other popup elements, check whether there are more links that can be enabled.